packages/toolkit/scripts/test/launchTestUtilities.ts

```diff
@@ -64,7 +64,7 @@ async function getVSCodeCliArgs(params: {
     let disableExtensionsArgs: string[] = []
     let disableWorkspaceTrustArg: string[] = []
 
-    if (params.suite !== 'unit') {
+    if (params.suite === 'integration' || params.suite === 'e2e') {
         disableExtensionsArgs = await getCliArgsToDisableExtensions(params.vsCodeExecutablePath, {
```

Comment:
Should we pass `--disable-extensions` (i.e disable _all_ extensions) for the unit/web cases?

Reply:
Makes sense, this was defined in the launch.json but not here. Added it in below